1.1/MetadataGranularities/README.md

```diff
@@ -0,0 +1,200 @@
+# Metadata Granularities
+
+This sample demonstrates the use of [Metadata in 3D Tiles 1.1](https://github.com/CesiumGS/3d-tiles/blob/b7fa517e88149942bc6166b11944bac67565369e/specification/README.md#metadata) on different levels of granularity. The sample consists of a tileset with 4 tiles where each tile has 5 contents, and the contents are assigned to two different groups. Metadata is assigned to the tileset, the tiles, each content, and to the groups.
+
+## Screenshot
+
+![MetadataGranularities](screenshot/MetadataGranularities.png)
+
+## Sandcastle Code
+
+```JavaScript
+var viewer = new Cesium.Viewer('cesiumContainer');
+
+// Create the tileset, and set its model matrix to move it
+// to a certain position on the globe
+var tileset = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
+    url : 'http://localhost:8003/1.1/MetadataGranularities/tileset.json',
+    maximumScreenSpaceError: 1,
```

Comment:
Use the default `maximumScreenSpaceError` if possible, which would mean modifying the geometric error in the tileset. I would be afraid of someone copying the code and using `maximumScreenSpaceError: 1` in their application without understanding the implications.